src/tests/ut/visit/visit_test.hpp

```diff
@@ -5,7 +5,7 @@
 
 class Doctor;
 
-struct AssociationTest : ::testing::Test
+struct VisitTestsFixture : ::testing::Test
```

Comment:
please switch to class and use `protected`

Reply:
Ah, I meant a protected keyword inside as you did. Inheritance has to be public, well done :)